bundle/Generator/ConfigurationGenerator.php

```diff
@@ -16,6 +17,15 @@
 
 class ConfigurationGenerator extends Generator
 {
+    public const LOCAL_DESIGN = 'local';
+    public const REMOTE_DESIGN = 'remote';
+
+    public function __construct(ContainerInterface $container)
```

Comment:
Is the constructor needed here? Isn't the parent called implicitly?

Reply:
Ah this was leftover from development. Will remove!

Reply:
Removed in 126d4c0d7fff52818b99c8cd52134bf9888a162a